chapters/fa/glossary/1.mdx

```diff
@@ -1,151 +1,153 @@
 <div dir="rtl">
 # واژه‌نامه
 
-| معادل در منبع         | معادل در ترجمه   |
```

Comment:
I get what you did here and want to thank you for the effort and the attention to detail Saeid jan. With the initial set of words I used an online service that automatically used a calculated fixed width in markup but then as we added new words and phrases we couldn't keep up that width, as will happen again in future with this version as well. I also have multiple diverging branches for pages half-done, that add lines to this file in arbitrary places that I don't really remember, and I'm sure other collaborators do as well. If we do this now, diffing becomes a manual task for us and one that is very time-consuming now that we are at 100+ words, as evidenced by the fact that right here I can't really tell what words you added, and neither can the Github diff. Please revert this to the old version and only add the new words. That version will greatly reduce the overhead for other collaborators when diffing.

That ofc holds true only for the beginning of the file to the end of the long word list. From then on please go ahead and apply what ever formatting you think is cool. In fact, while you're at it, if you want, remove the English footnote at the end. I think we are all on-board with our equivalent for inference now, and this line in English, no longer serves any purpose.

If the only word you added is "collaborative filtering" at the end, please capitalize.